page.js

```diff
@@ -0,0 +1,9 @@
+function webpage(){
+	var name = document.getElementById("name");
+
+	var sentence = "Hello " + name.value;
+
+	var output = document.getElementById("line");
+	output.textContent = sentence;
+}
```

Comment:
So, in the end, your function will look like this once you use jQuery and ES6 syntax:

```javascript
$("button").click(() => {
  const name = $("#name").val();

  const sentence = `Hello ${name}`;

  $("#line").html(sentence);
})
```